spec/system/reward_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Reward check', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+  let!(:reward) { create(:reward) }
+
+  it 'crud Reward for employee' do
```

Comment:
it's not crud, since we are testing index and show only :)

Reply:
I changed the name of this spec